src/main/java/org/gitlab4j/api/models/ApprovalRule.java

```diff
@@ -18,9 +18,6 @@
     private List<User> users;
     private List<Group> groups;
     private Boolean containsHiddenGroups;
-
-    @JsonSerialize(using = JacksonJson.UserListSerializer.class)
-    @JsonDeserialize(using = JacksonJson.UserListDeserializer.class)

```

Comment:
Should also remove the unused imports.

Reply:
Yes, good catch. I have optimized imports with IntelliJ now.